modules/rosa-upgrading-4_7_to_4_8_prerequisites.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * rosa_upgrading/rosa-upgrading-sts.adoc
+
+[id="rosa-upgrading-4-7-to-4-8-prerequisites_{context}"]
+= Prerequisites for upgrading from 4.7 to 4.8
+
+This section details the steps that are required before upgrading a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS) from version 4.7 to 4.8. Before upgrading, you must meet the following prerequisites:
+
+* Update the AWS Identity and Access Management (IAM) account-wide roles and policies, including the Operator policies to version 4.8.
+* After updating the roles and policies, you must update the value of the `cloudcredential.openshift.io/upgradeable-to` annotation in the `CloudCredential` custom resource to `v4.8`. This indicates that the cluster is ready to upgrade.
+
+.Prerequisites
+
+* You have installed and configured the latest AWS, ROSA, and `oc` CLIs on your installation host.
+* You have the permissions required to update the AWS account-wide roles and policies.
+* You have access to the cluster as a user assigned to the `cluster-admins` group.
+
+.Procedure
+
+. Update the account-wide roles and policies, including the Operator policies, to version 4.8:
++
+[source,terminal]
+----
+$ rosa create account-roles --mode auto --version 4.8
+----
++
+[IMPORTANT]
+====
+If you used a custom prefix when you created the roles and policies for version 4.7, you must include the `--prefix` option and specify the same prefix name. This ensures that the existing roles and policies used by the cluster are updated.
```

Comment:
nit: I'm unsure if "phrasal verb" is the correct term, but "you used a" and the later verb "created" catches my eye.  Perhaps "If you created the roles and policies for version 4.7 with a custom prefix, you must include..."

nit: Second sentence: same "this" nit as preceding comment, maybe "Specifying the prefix name ensures..."  (Not sure of the correct noun,)

Reply:
Thanks Mike, I will implement these suggestions.

Out of curiosity, what is wrong with using the word "this" to follow on from something explained in the previous sentence?